contrib/build-wine/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+# Lucky number
+export PYTHONHASHSEED=22
+
+here=$(dirname "$0")
+
+echo "Clearing $here/build and $here/dist..."
+rm $here/build/* -rf
```

Comment:
```
# here=$(dirname "$0")
# ...
# rm $here/build/* -rf
# rm $here/dist/* -rf
```

This script looks dangerous. If user does not have coreutils or if there is another dirname executable in the local bin directory - this script can remove /build/* and /dist/* in the root of user's filesystem.

```
$ here=$(nothing "$0")
$ echo rm $here/build/* -rf
$ echo rm $here/dist/* -rf
```

We have to check that _${here}_ is not empty at least. Also do we really need it or can we just use _${PWD}_?